wallet-services/src/main/java/io/meeds/wallet/service/WalletAccountServiceImpl.java

```diff
@@ -569,6 +570,13 @@ public void switchWalletProvider(long identityId,
     boolean isNew = !isUserWalletInitialized(wallet.getId()) && !accountStorage.hasWalletBackup(identityId) && (existingWallet == null || StringUtils.isBlank(existingWallet.getAddress()) || StringUtils.isNotBlank(existingWallet.getInitializationState()));
     String eventName = isNew ? NEW_ADDRESS_ASSOCIATED_EVENT
                              : MODIFY_ADDRESS_ASSOCIATED_EVENT;
+    try {
+      Map<String, String> walletDetails = new HashMap<>();
+      walletDetails.put("walletAddress", wallet.getAddress());
+      getListenerService().broadcast(EVM_HOLD_ACTION_EVENT, walletDetails, "");
```

Comment:
From wallet, broadcast a specific event in destination of EVM addon ? Can you please rework this to not make such a change